inference-engine/ie_bridges/python/sample/speech_sample/speech_sample.py

```diff
@@ -0,0 +1,390 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+import argparse
+import logging as log
+import re
+import sys
+from timeit import default_timer
+from typing import Any, IO
+
+import numpy as np
+from openvino.inference_engine import ExecutableNetwork, IECore
+
+
+def parse_args() -> argparse.Namespace:
+    """Parse and return command line arguments"""
+    parser = argparse.ArgumentParser(add_help=False)
+    args = parser.add_argument_group('Options')
+    model = parser.add_mutually_exclusive_group(required=True)
+
+    args.add_argument('-h', '--help', action='help', help='Show this help message and exit.')
+    model.add_argument('-m', '--model', type=str,
+                       help='Path to an .xml file with a trained model (required if -rg is missing).')
+    model.add_argument('-rg', '--import_gna_model', type=str,
+                       help='Read GNA model from file using path/filename provided (required if -m is missing).')
+    args.add_argument('-i', '--input', required=True, type=str, help='Required. Path to an utterance file.')
+    args.add_argument('-o', '--output', type=str, help='Optional. Output file name to save inference results.')
+    args.add_argument('-r', '--reference', type=str,
+                      help='Optional. Read reference score file and compare scores.')
+    args.add_argument('-d', '--device', default='CPU', type=str,
+                      help='Optional. Specify a target device to infer on. '
+                      'CPU, GPU, MYRIAD, GNA_AUTO, GNA_HW, GNA_SW_FP32, GNA_SW_EXACT and HETERO with combination of GNA'
+                      ' as the primary device and CPU as a secondary (e.g. HETERO:GNA,CPU) are supported. '
+                      'The sample will look for a suitable plugin for device specified. Default value is CPU.')
+    args.add_argument('-bs', '--batch_size', default=1, type=int, help='Optional. Batch size 1-8 (default 1).')
+    args.add_argument('-qb', '--quantization_bits', default=16, type=int,
+                      help='Optional. Weight bits for quantization: 8 or 16 (default 16).')
+    args.add_argument('-wg', '--export_gna_model', type=str,
+                      help='Optional. Write GNA model to file using path/filename provided.')
+    # TODO: Find a model that applicable for -we option
+    args.add_argument('-we', '--export_embedded_gna_model', type=str, help=argparse.SUPPRESS)
+    # TODO: Find a model that applicable for -we_gen option
```

Comment:
TODO means that these options are not validated?

Reply:
it is hidden option and doesn't validated

Reply:
@generalova-kate but sample code is not hidden and this TODO is visible for everyone in code)

Reply:
we will remove it